extensions/support-page/main.ts

```diff
@@ -0,0 +1,23 @@
+import { LensMainExtension, Registry, windowManager } from "@k8slens/extensions";
+import { supportPageURL } from "./src/support.route";
+
+export default class SupportPageMainExtension extends LensMainExtension {
+  async onActivate() {
+    console.log("support page extension activated")
+  }
+
+  async registerAppMenus(registry: Registry.MenuRegistry) {
+    this.disposers.push(
+      registry.add({
+        parentId: "help",
+        label: "Support",
```

Comment:
So need to remove from that place? What is the question / propose?

Reply:
Probably remove this "Support" menu item (although it's nice to demonstrate that this can be done). The existing menu items have the advantage of taking you directly to the link in one click, whereas with "Support" it's one click to get to the support page then another to get to the link.